terratest/src/test/deploy_test.go

```diff
@@ -0,0 +1,160 @@
+package test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	//k8sresource "k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+)
+
+const PegaHelmChartPath = "../../../charts/pega"
+
+// set action execute to install
+var options = &helm.Options{
+	SetValues: map[string]string{
+		"global.actions.execute": "deploy",
+	},
+}
+
+func TestPegaStandardTierDeployment(t *testing.T) {
```

Comment:
General feedback for all of these test files - can you please add a brief comment to each one to describe what is being tested and what the expectation is?

Reply:
Fixed